plugins/modules/ec2_vpc_peer.py

```diff
@@ -224,7 +224,19 @@
 vpc_peering_connection:
   description: The details of the VPC peering connection as returned by Boto3 (snake cased).
   returned: success
-  type: dict
+  type: complex
+  contains:
+    vpc_peering_connection_id:
+      type: str
+      sample: pcx-034223d7c0aec3cde
+    accepter_vpc_info:
+      type: dict
+    requester_vpc_info:
+      type: dict
+    tags:
+      type: dict
+    status:
+      type: dict
```

Comment:
You might be able to copy&paste a more complete version of this from ec2_vpc_peering_info

Reply:
Good point. Anyhow wasn't sure how far down the nesting I should really go...

Reply:
The advantage of getting reasonably specific is that you can tie it to the tests you add.  I personally find it frustrating when the return docs just say "it's a complex dict", and I have to play around and see what it actually returns.